src/java/org/apache/cassandra/index/sasi/analyzer/filter/StemmerFactory.java

```diff
@@ -64,24 +64,24 @@
     static
     {
         SUPPORTED_LANGUAGES = new HashMap<>();
-        SUPPORTED_LANGUAGES.put("de", germanStemmer.class);
-        SUPPORTED_LANGUAGES.put("da", danishStemmer.class);
-        SUPPORTED_LANGUAGES.put("es", spanishStemmer.class);
-        SUPPORTED_LANGUAGES.put("en", englishStemmer.class);
-        SUPPORTED_LANGUAGES.put("fl", finnishStemmer.class);
-        SUPPORTED_LANGUAGES.put("fr", frenchStemmer.class);
-        SUPPORTED_LANGUAGES.put("hu", hungarianStemmer.class);
-        SUPPORTED_LANGUAGES.put("it", italianStemmer.class);
-        SUPPORTED_LANGUAGES.put("nl", dutchStemmer.class);
-        SUPPORTED_LANGUAGES.put("no", norwegianStemmer.class);
-        SUPPORTED_LANGUAGES.put("pt", portugueseStemmer.class);
-        SUPPORTED_LANGUAGES.put("ro", romanianStemmer.class);
-        SUPPORTED_LANGUAGES.put("ru", russianStemmer.class);
-        SUPPORTED_LANGUAGES.put("sv", swedishStemmer.class);
-        SUPPORTED_LANGUAGES.put("tr", turkishStemmer.class);
+        SUPPORTED_LANGUAGES.put("de", GermanStemmer.class);
+        SUPPORTED_LANGUAGES.put("da", DanishStemmer.class);
+        SUPPORTED_LANGUAGES.put("es", SpanishStemmer.class);
+        SUPPORTED_LANGUAGES.put("en", EnglishStemmer.class);
+        SUPPORTED_LANGUAGES.put("fl", FinnishStemmer.class);
+        SUPPORTED_LANGUAGES.put("fr", FrenchStemmer.class);
+        SUPPORTED_LANGUAGES.put("hu", HungarianStemmer.class);
+        SUPPORTED_LANGUAGES.put("it", ItalianStemmer.class);
+        SUPPORTED_LANGUAGES.put("nl", DutchStemmer.class);
+        SUPPORTED_LANGUAGES.put("no", NorwegianStemmer.class);
+        SUPPORTED_LANGUAGES.put("pt", PortugueseStemmer.class);
+        SUPPORTED_LANGUAGES.put("ro", RomanianStemmer.class);
+        SUPPORTED_LANGUAGES.put("ru", RussianStemmer.class);
+        SUPPORTED_LANGUAGES.put("sv", SwedishStemmer.class);
+        SUPPORTED_LANGUAGES.put("tr", TurkishStemmer.class);
```

Comment:
No `PolishStemmer`. I'm disappointed ;)

Reply:
Talk with sasi

Reply:
Seriously though, if a third-party implemented a `PolishStemmer`, would it be possible now to plug one in? 
Or are we limited only to those stemmers above?

Reply:
https://github.com/apache/lucene/search?q=polish